src/compound_io.rs

```diff
@@ -0,0 +1,64 @@
+use codec::{Decode, Encode};
+use gstd::{ActorId, TypeInfo};
+
+#[derive(Debug, Decode, Encode, Clone, TypeInfo)]
+pub struct CompoundInit {
+    pub token_address: ActorId,
+    pub ctoken_address: ActorId,
+    pub interest_rate: u128,
+    pub collateral_factor: u128,
+    pub borrow_rate: u128,
+}
+
+#[derive(Debug, Decode, Encode, Clone, TypeInfo)]
+pub enum CompoundAction {
+    LendTokens { amount: u128 },
+    BorrowTokens { amount: u128 },
+    RefundTokens { amount: u128 },
+    WithdrawTokens { amount: u128 },
+    Balance { address: ActorId },
```

Comment:
We can use tuples here where it is intuitive.
```suggestion
    LendTokens(u128),
    BorrowTokens(u128),
    RefundTokens(u128),
    WithdrawTokens(u128),
    BalanceOf(ActorId),
```